src/components/Category.js

```diff
@@ -1,7 +1,25 @@
 import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { checkStatusAction } from "../redux/categories/categories";
 
 function Category() {
-  return <h3>Under Construction</h3>;
+  const categoryStore = useSelector((state) => state.categoriesReducers);
+  const dispatch = useDispatch();
+
+  const checkStatus = () => {
+    dispatch(checkStatusAction);
+  };
+  return (
+    <button
+      type="button"
+      onClick={() => {
+        checkStatus();
+      }}
+    >
```

Comment:
- [ ] Make sure when `checkStatus` button is clicked  a specific message  is printed. 
![Screenshot from 2022-05-05 15-10-08](https://user-images.githubusercontent.com/65860678/166930730-097366a8-3a6f-4292-b583-00a3da0fb150.png)